<launch>
  <!-- Map server -->
  <param name="use_sim_time" value="true"/>
  <arg name="map_name"/>
  <arg name="scan_topic" default="stable_scan"/>
  <node name="map_server" pkg="map_server" type="map_server" args="$(find robot_localizer)/maps/$(arg map_name).yaml"/>
  <node name="bag_player" pkg="rosbag" type="play" args="$(find robot_localizer)/bags/$(arg map_name).bag --clock --pause" output="screen"/>

  <include file="$(find neato_node)/launch/set_urdf.launch"/>

  <!-- Localization -->
  <node name="pf" pkg="robot_localizer" type="pf.py" output="screen">
    <remap from="scan" to="$(arg scan_topic)"/>
  </node>
</launch>
